docker-compose.yml

```diff
@@ -60,7 +65,7 @@ services:
     image: timothyb89/monasca-sidecar:1.0.0
 
   monasca:
-    image: monasca/api:master-prometheus
+    image: monasca/api:latest
```

Comment:
`monasca/api:master-20170524-151708` was just pushed and would be the correct API image to use